src/main/java/uk/gov/justice/hmpps/prison/repository/sql/AdjudicationsRepositorySql.kt

```diff
@@ -31,17 +31,17 @@ enum class AdjudicationsRepositorySql(val sql: String) {
         ,       OO.DESCRIPTION      DESCRIPTION
                 FROM (
                         SELECT DISTINCT AIC.CHARGED_OIC_OFFENCE_ID AS OFFENCE_IDS
-                                FROM AGENCY_INCIDENTS AI
+                        FROM AGENCY_INCIDENTS AI
                                 INNER JOIN AGENCY_INCIDENT_PARTIES AIP ON AIP.AGENCY_INCIDENT_ID = AI.AGENCY_INCIDENT_ID
                                 INNER JOIN AGENCY_LOCATIONS AL ON AL.AGY_LOC_ID = AI.AGY_LOC_ID
                                 INNER JOIN AGENCY_INCIDENT_CHARGES AIC ON AIC.AGENCY_INCIDENT_ID = AIP.AGENCY_INCIDENT_ID AND AIC.PARTY_SEQ = AIP.PARTY_SEQ
-                                WHERE AIP.OFFENDER_BOOK_ID IN (
-                                SELECT OFFENDER_BOOK_ID FROM OFFENDER_BOOKINGS OB
-                                        JOIN OFFENDERS OFF ON OFF.OFFENDER_ID = OB.OFFENDER_ID
-                                        WHERE OFF.OFFENDER_ID_DISPLAY = :offenderNo)
+                                INNER JOIN OFFENDER_BOOKINGS OB ON OB.OFFENDER_BOOK_ID = AIP.OFFENDER_BOOK_ID
+                                INNER JOIN OFFENDERS OFF ON OFF.OFFENDER_ID = OB.OFFENDER_ID
+                        WHERE OFF.OFFENDER_ID_DISPLAY = :offenderNo
+                              AND OB.BOOKING_SEQ = 1
```

Comment:
Used this `AND OB.BOOKING_SEQ = 1` to determine latest booking

Reply:
Should we be checking for active bookings too?

Reply:
That's convenient - lucky you

Reply:
@JamieDraperUK the ticket  only says the latest booking - but will check with John/Dom

Reply:
@JamieDraperUK checked with Dom - it is the latest booking they want here